packages/analytics/src/public-types.ts

```diff
@@ -288,4 +288,14 @@ export interface EventParams {
   page_path?: string;
   [key: string]: unknown;
 }
+
+/** Maps the applicable end user consent state. */
+export interface ConsentSettings {
+  ad_storage?: ConsentStatusString;
```

Comment:
Add documentation above each option, can steal it from here https://developers.google.com/tag-platform/tag-manager/templates/consent-apis#consent_state_and_consent_types

Reply:
Added external doc link